src/background.jl

```diff
@@ -0,0 +1,64 @@
+"""
+    gen_img = generatecanvas(colortype, x, y)
+    gen_img = generatecanvas(colortype, x, y, background)
+    gen_img = generatecanvas(colortype, [x,y])
+    gen_img = generatecanvas(colortype, [x,y], background)
+
+    Creates an image of with the given colortype and size.
+    Defaults to a black solid image
+"""
+generatecanvas(colortype::Type, x::Int, y::Int) = zeros(colortype, (x,y))
```

Comment:
Since the work is being done by `zeros`, why do we effectively have to create an alias to it?